template/src/store/saga/app.ts

```diff
@@ -1,9 +1,10 @@
-import {takeLatest, call, put} from 'redux-saga/effects'
-import {appActions} from '../reducers'
+import {AnyAction} from 'redux'
+import {call, put, takeLatest} from 'redux-saga/effects'
 import RouteKey from '../../navigation/RouteKey'
 import {getData} from '../../utilities/storage'
+import {appActions} from '../reducers'
 
-function* getAppSettingSaga(): IterableIterator<void> {
+function* getAppSettingSaga(): IterableIterator<AnyAction> {
   try {
     const token = yield call(getData, 'token')
```

Comment:
Should store another or signature key name instead of the name easy to guess

Reply:
I.don't get your point. Can you explain in detail ?